packages/@lwc/rollup-plugin/README.md

```diff
@@ -12,17 +12,20 @@ yarn add --dev @lwc/rollup-plugin
 
 ```js
 // rollup.config.js
-import rollupPlugin from '@lwc/rollup-plugin';
+import lwc from '@lwc/rollup-plugin';
 
 export default {
     input: './src/main.js',
-    plugins: [rollupPlugin()],
+    plugins: [lwc()],
 };
 ```
 
 ## Options
 
--   `rootDir` (string, optional, default: `input directory`) - set the LWC module directory
--   `sourcemap` (boolean, optional, default: `false`) - make the LWC compiler produce source maps
--   `modules` Mapping of module specifiers.
--   `stylesheetConfig` (object, optional, default: `{}`) - the configuration to pass to the `@lwc/style-compiler`
+-   `include` (type: `string | string[]`, default: `null`) - A [minimatch pattern](https://github.com/isaacs/minimatch), or array of patterns, which specifies the files in the build the plugin should transform on. By default all files are targeted.
```

Comment:
are these new or weren't documented?

Reply:
Those were not documented so I added them.